prisma/schema.prisma

```diff
@@ -44,4 +44,9 @@ model MitmachWoche {
   isProduction Boolean
   event        String   @db.VarChar(24)
   timestamp    DateTime @default(now())
+}
+
+model PrototypeThreadStatus {
+  threadId     String @db.VarChar(24) @id
+  status       String @db.VarChar(24)
```

Comment:
easiest db migration ever 👍

Reply:
creating a database is nothing scary anymore :)